### PR TITLE
Add installation parameters for Mastodon

### PR DESCRIPTION
**Pull request by andrew-codechimp**
## Proposed change

Add installation parameters for Mastodon

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new configuration section for the Mastodon integration, enhancing setup instructions with required fields: `URL`, `Client key`, `Client secret`, and `Access token`.
- **Documentation**
	- Improved clarity and completeness of integration instructions for users to configure the Mastodon integration effectively.

### CODE REVIEW

File: source/_integrations/mastodon.markdown [hunk 1]
```diff
@@ -26,6 +26,25 @@ If you want to grant only required accesses, uncheck all checkboxes then check o
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+URL:
+  description: The URL of your Mastodon instance, for example `https://mastodon.social`.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/mastodon.markdown]:
```suggestion
```

File: source/_integrations/mastodon.markdown [hunk 2]
```diff
@@ -26,6 +26,25 @@ If you want to grant only required accesses, uncheck all checkboxes then check o
...
+Client key:
+  description: The client key for the application created within your Mastodon account web interface.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/mastodon.markdown]:
```suggestion
```

File: source/_integrations/mastodon.markdown [hunk 3]
```diff
@@ -26,6 +26,25 @@ If you want to grant only required accesses, uncheck all checkboxes then check o
...
+Client secret:
+  description: The client secret for the application created within your Mastodon account web interface.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/mastodon.markdown]:
```suggestion
```

File: source/_integrations/mastodon.markdown [hunk 4]
```diff
@@ -26,6 +26,25 @@ If you want to grant only required accesses, uncheck all checkboxes then check o
...
+Access token:
+  description: The access token for the application created within your Mastodon account web interface.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/mastodon.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]
Those aren't used by `configuration_basic`

../Frenck

File: source/_integrations/mastodon.markdown (hunk 4 above)
Comment (andrew-codechimp) [reply source/_integrations/mastodon.markdown]: Trust me to copy one that has them (Cambridge Audio and a few others)
I'll fix.

**Review by frenck — APPROVED** [review]
Thanks, @andrew-codechimp 👍 

../Frenck

**Pull request merged**